test/response/status.js

```diff
@@ -7,59 +7,53 @@ const statuses = require('statuses');
 const assert = require('assert');
 const Koa = require('../..');
 
-describe('res.status=', function(){
-  describe('when a status code', function(){
-    describe('and valid', function(){
-      it('should set the status', function(){
+describe('res.status=', () => {
+  describe('when a status code', () => {
+    describe('and valid', () => {
+      it('should set the status', () => {
         const res = response();
         res.status = 403;
         res.status.should.equal(403);
       });
 
-      it('should not throw', function(){
-        assert.doesNotThrow(function(){
+      it('should not throw', () => {
+        assert.doesNotThrow(() => {
           response().status = 403;
         });
       });
     });
 
-    describe('and invalid', function(){
-      it('should throw', function(){
-        assert.throws(function(){
+    describe('and invalid', () => {
+      it('should throw', () => {
+        assert.throws(() => {
           response().status = 999;
         }, 'invalid status code: 999');
       });
     });
 
-    describe('and custom status', function(){
-      before(function(){
-        statuses['700'] = 'custom status';
-      });
+    describe('and custom status', () => {
+      before(() => statuses['700'] = 'custom status');
 
-      it('should set the status', function(){
+      it('should set the status', () => {
         const res = response();
         res.status = 700;
         res.status.should.equal(700);
       });
 
-      it('should not throw', function(){
-        assert.doesNotThrow(function(){
-          response().status = 700;
-        });
+      it('should not throw', () => {
+        assert.doesNotThrow(() => response().status = 700);
       });
     });
   });
 
-  describe('when a status string', function(){
-    it('should throw', function(){
-      assert.throws(function(){
-        response().status = 'forbidden';
-      }, 'status code must be a number');
+  describe('when a status string', () => {
+    it('should throw', () => {
+      assert.throws(() => response().status = 'forbidden', 'status code must be a number');
```

Comment:
do we even have a 80 character limit? if it's not an eslint rule, it doesn't matter!